docs/docs/community/maintainers/triage-issues.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Issue triaging
+description: Guidelines and tips for maintaining Nebari
+---
+
+Issue triage is a process by which Nebari maintainers and contributors intake and reviews new GitHub issues and requests,
+and organize them to be actioned. Triaging involves categorizing issues and Pull Requests based on factors such as priority,
+area of ownership of the issue (tags), and the issue kind (bug, feature, and so on).
+
+Triage can happen asynchronously and continuously, or in regularly scheduled meetings.
+
+## Why Is Triaging Beneficial?
```

Comment:
```suggestion
## Why Is triaging beneficial?
```
To use sentence case for headings. :)

Reply:
I kept mixing sentence and title case 😶‍🌫️  sorry